packages/lib-classifier/src/hooks/usePanoptesUser.js

```diff
@@ -20,5 +42,5 @@ export default function usePanoptesUser() {
     }
   }, [authClient])
 
-  return user
+  return { user, loading }
```

Comment:
```suggestion
  return { user, isLoading: loading }
```
Change this for consistency with `useSWR`.